### PR TITLE
Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by afiamoah**
- Documentations are done in a  professional way

-  A new GitHub repo was created.

- A new branch (Main-Page) was created according to GitHub Flow.

-  a .gitignore file was created successfully

- Html and css linters were configured with no errors.
- Node_Modules  were ignored while comitting  changes.  
- 
-A Descriptive Readme file was added.

Added html file that includes a header with the text "Hello Microverse!".

A css file that includes one class with styles for the header was added

A test.md file with the text "This file should be ignored by git". was added but was ignored in .gitgnore file
.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,4 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (omarsalem7) [inline README.md:1]: - [ ] Kindly use [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) and customize it to make your readme professional.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @afiamoah 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️

- [x] Kindly make sure your Pull Request should have a descriptive title telling you what changes are inside.
- [x] Kindly make sure your Pull Request should have a summary describing introduced changes in more detail.
- [x]  _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by afiamoah** [comment]
than you for the review

File: README.md [hunk 2]
```diff
@@ -1,25 +1,4 @@
...
 
-#Microverse first Project
-
```
Comment (roshan-bajgain) [inline README.md:4]: - [x] Kindly add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project as requested by previous reviewer

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @afiamoah, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (afiamoah) [reply README.md:4]: #Project Name
 Set up a "Hello Microverse" project"

## Project Description
This project was created to as a Microverse Hello world project.this project uses linters to check code errors and fix bugs as well as uses github flow to ensure best practices.


## Built With

- HTML & CSS

## Authors

👤 **Author1**

- GitHub: [@githubhandle](https://github.com/afiamoah)
- Twitter: [@twitterhandle](https://twitter.com/afiamoah)
- LinkedIn: [LinkedIn](https://linkedin.com/in/afiamoah)


## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

## Show your support

Give a ⭐️ if you like this project!

## Acknowledgments

- Hat tip to anyone whose code was used
- Inspiration

**Pull request merged**